src/dwarf/units.rs

```diff
@@ -498,6 +509,37 @@ mod tests {
         });
     }
 
+    /// Benchmark the parsing of inlined function information, end-to-end.
```

Comment:
please include your local benchmark results into the commit details when you are done, I'd be curious to see the relative performance. Thanks!

Reply:
Will do.